apps/site/pages/en/learn/asynchronous-work/discover-javascript-timers.md

```diff
@@ -33,7 +33,7 @@ const myFunction = (firstParam, secondParam) => {
 setTimeout(myFunction, 2000, firstParam, secondParam);
 ```
 
-`setTimeout` returns the timer id. This is generally not used, but you can store this id, and clear it if you want to delete this scheduled function execution:
+`setTimeout` returns a timer identifier. In Node.js, this is a Timeout object that provides additional functionality, while in browsers it returns a numeric ID. You can store this identifier and use it to cancel the scheduled function execution:
 
 ```js
 const id = setTimeout(() => {
```

Comment:
These example should probably not use `id`, they are meant for Node.js, so let's use a more specific variable name

```suggestion
const timeout = setTimeout(() => {
```